docs/grpc/docs.md

```diff
@@ -1072,6 +1072,7 @@ Note: 1kB = 1 vector of size 256. |
 | Prefix | 1 |  |
 | Whitespace | 2 |  |
 | Word | 3 |  |
+| Charabia | 4 |  |
```

Comment:
I think my first concern is the naming of the API, surfacing `charabia` seems like an implementation details leak.

What do you think about `multilingual` instead?

Reply:
right, this was my concern as well -- and `multilingual` is much better.